### ISSUE TITLE
Can't install NPM extensions

### ISSUE THREAD

**Issue by jtovar43** [theia-ide/theia#5991]
### Description
When trying to install npm extensions via the terminal I get errors. The most common one is `Failed at the nsfw@1.2.5 install script.` I am trying to get Theia to support code completion and linting for Python but I can't find much support on how to do so. Is installing NPM extensions the correct way to do so?

**Comment by akosyakov** [comment]
@jtovar43 does it fail sporadically for you? Or it is consistent and with the same error? Maybe npm registry had troubles or your network connection?

**Comment by jtovar43** [comment]
Its usually consistent with that same error. I don't think it's an issue with the network.

**Comment by kittaakos** [comment]
> I am trying to get Theia to support code completion and linting for Python



> Is installing NPM extensions the correct way to do so?

No. See the Python example: https://github.com/theia-ide/theia-apps/blob/0471d87fb1312982689490d6992cc6b32f167f3a/theia-python-docker/latest.package.json#L99
